tests/test_utils/test_schemaview.py

```diff
@@ -641,62 +674,82 @@ def test_direct_remote_imports_additional() -> None:
     assert len(view.all_classes()) > 0
 
 
-def test_import_map() -> None:
-    """Path to import file should be configurable."""
-    for im in [{"core": "/no/such/file"}, {"linkml:": "/no/such/file"}]:
-        view = SchemaView(SCHEMA_WITH_IMPORTS, importmap=im)
-        with pytest.raises(FileNotFoundError):
-            view.all_classes()
-
-    for im in [None, {}, {"core": "core"}]:
-        view = SchemaView(SCHEMA_WITH_IMPORTS, importmap=im)
+@pytest.mark.parametrize("importmap", [{"core": "/no/such/file"}, {"linkml:": "/no/such/file"}])
+def test_import_map_fail(importmap: dict[str, Any]) -> None:
+    view = SchemaView(SCHEMA_WITH_IMPORTS, importmap=importmap)
+    with pytest.raises(FileNotFoundError, match="No such file or directory:"):
         view.all_classes()
-        assert (
-            view.imports_closure().sort() == ["kitchen_sink", "core", "linkml:types"].sort()
-        )  # Assert imports closure
-        assert ACTIVITY in view.all_classes()  # Assert ACTIVITY is in all classes
-        assert ACTIVITY not in view.all_classes(imports=False)  # Assert ACTIVITY is not in classes without imports
 
 
-def test_mergeimports() -> None:
+@pytest.mark.parametrize("importmap", [None, {}, {"core": "core"}, {"core": str(INPUT_DIR_PATH / "core")}])
+def test_import_map(importmap: dict[str, Any]) -> None:
+    """Path to import file should be configurable."""
+    view = SchemaView(SCHEMA_WITH_IMPORTS, importmap=importmap)
+    view.all_classes()
+    # ensure that all imports have loaded
+    assert set(view.imports_closure()) == {"kitchen_sink", "core", "linkml:types"}
+    # ensure that ACTIVITY only appears in all_classes if imports are enabled
+    assert ACTIVITY in view.all_classes()
+    assert ACTIVITY not in view.all_classes(imports=False)
+
+
+def test_merge_imports_kwargs(schema_view_with_imports: SchemaView, sv_merged_imports_keyword: SchemaView) -> None:
```

Comment:
this is `test_mergeimports` from the left -- added some comments and extra assertions